katdal/sensordata.py

```diff
@@ -296,11 +296,11 @@ def root(self):
         return TelstateToStr(self._telstate.root())
 
     def keys(self, filter='*'):
-        return to_str(self._telstate.keys(filter))
+        return self._telstate.keys(filter)
```

Comment:
This whole function can be deleted, because the version in `_telstate` will work (and will be found by `__getattr__`).